### DISCUSSION TITLE
history

### DISCUSSION THREAD

**Discussion by MatthewAJohnson281** [Monika-After-Story/MonikaModDev#9454] [QA - code and content creation (technical)]
Where can you find the full conversation history? I need to copy it for a friend who wants to make Monika AI .

**Comment by Wingdinggaster656** [comment]
You mean where you can get all the dialogues, right? Then get it in the source code.
`script-topics.rpy` should fits most need, though other files include other dialogues.

**Comment by MatthewAJohnson281** [comment]
Are they just in order?

**Comment by MatthewAJohnson281** [comment]
I mean like where the history is, it shows the full conversation. I cant highlight that though.

**Comment by Booplicate** [comment]
If you indeed meant history, then you can access it by pressing `Esc` -> `History`. If you mean source files, then use GitHub.

**Reply by MatthewAJohnson281** [reply]
it wont let me copy the history. and the source code is a mess so I decided to use all of Monika's original dialogue so the architecture can have a reference to the way she talks.

**Comment by MatthewAJohnson281** [comment]
I figured it out I was on the backed up copy of the folder and it wasnt updated
